modules/reports/views/reports/header.php

```diff
@@ -17,15 +17,20 @@
 		<?php
 		// $report_time_formatted is already escaped
 		echo '<p>'._('Reporting period').': '.$report_time_formatted;
-		echo '<p>'._('Time zone').': '.$options['report_timezone'].' '.reports::timezone_utc_no($options['report_timezone']);
+		if ($type != 'logger') {
```

Comment:
I know this was also broken before - but don't we open more <p> tags than we close here?

Reply:
Do you mean the `<p>` tags? In that case, the closing tags can be omitted, see "tag omission" at  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Reply:
Aha... The more you know :+1: